app/controllers/posts_controller.rb

```diff
@@ -17,12 +17,8 @@ def edit
   end
 
   def create
-    p 'this is the params'
-    p post_params
-    p "this is the session id"
-    p session[:user_id]
     @post = Post.new(post_params)
-
+    @post.user_id = current_user.id
```

Comment:
I think it's possible to do `current_user.posts.new(post_params)` to create a new post, with the appropriate user id